CTP/shell/src/com/navercorp/cubridqa/shell/main/JdbcLocalTest.java

```diff
@@ -122,14 +135,26 @@ private void runTests(JdbcCaseMethodBean caseMethodBean){
 				failCaseCount++;
 			}
 			
-			res = caseClassPackageFullName + " => " + methodName  + " : " + (isSucc ? "OK" : "NOK");
+			res += caseClassPackageFullName + " => " + methodName  + " : " + (isSucc ? "OK" : "NOK");
 			long runTime = result == null ? 0 : result.getRunTime();
-			this.log.println(res + " => Elapse Time:"  + runTime);
+			this.runCaseLog.println(res + " => Elapse Time:"  + runTime);
 			if(failureMessage.length()>0) {
-				res += Constants.LINE_SEPARATOR + failureMessage;
-				this.log.println("Failure Message:" + Constants.LINE_SEPARATOR + failureMessage);
+					res += Constants.LINE_SEPARATOR + failureMessage;
+					this.runCaseLog.println("[Failure Message]:" + Constants.LINE_SEPARATOR + failureMessage);
+			}
+			System.out.flush();
+			System.err.flush();
+			if(baos.toString()!=null && baos.toString().trim().length() >0){
+				res += Constants.LINE_SEPARATOR + "Run_Log:"  + Constants.LINE_SEPARATOR + baos.toString();
 			}
-			context.getFeedback().onTestCaseStopEvent(caseClassPackageFullName + "=>" +  methodName, isSucc, runTime, res, "local", false, false, ingoreCount==0 ? Constants.SKIP_TYPE_NO : Constants.SKIP_TYPE_BY_INVALID_UNITCASE , 0);
+			this.runCaseLog.println(baos.toString());
+			this.runCaseLog.println("===================================================");
+			System.setOut(origStdout);
+			System.setErr(origStderr);
+			System.out.println(res);
+			System.out.println();
+			System.out.println("===================================================");
+			context.getFeedback().onTestCaseStopEvent(caseFile + ":" + methodName + "()", isSucc, runTime, res, "local", false, false, Constants.SKIP_TYPE_NO , 0);
```

Comment:
as is:
context.getFeedback().onTestCaseStopEvent(caseFile + ":" + methodName + "()", isSucc, runTime, res, "local", false, false, Constants.SKIP_TYPE_NO , 0);

to be:
if test is successful, make res to null.
We needn't record messages when a test case passed.